client/lib/importer/importer-config.js

```diff
@@ -0,0 +1,204 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { filter, orderBy, values } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import InlineSupportLink from 'components/inline-support-link';
+
+function getConfig( { siteTitle = '' } = {} ) {
+	const importerConfig = {};
+
+	importerConfig.wordpress = {
+		engine: 'wordpress',
+		type: 'file',
+		title: 'WordPress',
+		icon: 'wordpress',
+		description: translate(
+			'Import posts, pages, and media ' + 'from a WordPress export\u00A0file.'
```

Comment:
Not sure you need string concatenation here. Also, what is the `\u00A0` character for?

Reply:
I'd say it's there to prevent the word "file" from being orphaned at certain page widths, like this. It's probably OK.

<img width="624" alt="image" src="https://user-images.githubusercontent.com/1647564/62766556-a6e83600-ba8a-11e9-8082-65e9d9b0b962.png">

Reply:
That's correct. These were copied verbatim and only minimally changed from the originating components. Agree re: concatenation, I missed that... but I don't want to alter the strings and cause new untranslated strings  to appear. I think there are some issues with translation approach in general, in particular, `Need help {{inlineSupportLink/}}?` should be more like `Need help {{inlineSupportLink}}exporting your content{{/inlineSupportLink}}?`, as the former isn't translatable with the fragment, but those changes are outside of the scope of what I'm trying to accomplish in this PR.